src/routes/index.js

```diff
@@ -1,12 +1,14 @@
 import express from 'express'
 import {getTestRoutes} from './testRoutes'
-
+import {getSummoner} from './matchHistoryRoutes'
 
 
 function getRoutes() {
   const router = express.Router()
   router.use('/test', getTestRoutes())
+  router.use('/test-run', getSummoner())

```

Comment:
Instead of /test-run, consider something like this: 
```suggestion
  router.use('/tft', getTFTRoutes())

```